sys/crypto/aes.c

```diff
@@ -45,12 +45,14 @@
  */
 static const cipher_interface_t aes_interface = {
     AES_BLOCK_SIZE,
-    AES_KEY_SIZE,
```

Comment:
This must be considered as an API change which I would like to avoid here. My [comment](https://github.com/RIOT-OS/RIOT/pull/16183#issuecomment-796800999) was only meant for clarification, not to propose such a change.